common/components/org.wso2.carbon.analytics.util/pom.xml

```diff
@@ -84,13 +76,14 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Name>${project.artifactId}</Bundle-Name>
-                        <Private-Package>org.wso2.carbon.analytics.common.internal.*</Private-Package>
+                        <Private-Package>org.wso2.carbon.analytics.common.spark.internal.*</Private-Package>
```

Comment:
Later, we should move all internals to "org.wso2.carbon.analytics.common.internal.*"